Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -341,14 +346,29 @@ extension LoginListViewController: LoginDataSourceObserver {
 
 // MARK: - UITableViewDelegate
 extension LoginListViewController: UITableViewDelegate {
-
     func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
         // Force the headers to be hidden
-        return 0
+        return section == 1 ? 44 : 0
```

Comment:
This comment is wrong. One header is shown at the start of the first logins section.